test/handlers/CompatibilityFallbackHandler.spec.ts

```diff
@@ -131,6 +150,24 @@ describe("CompatibilityFallbackHandler", async () => {
             const sig2 = await signHash(user2, calculateSafeMessageHash(validator,dataHash, await chainId()))
             expect(await validator.callStatic['isValidSignature(bytes32,bytes)'](dataHash, buildSignatureBytes([sig1, sig2]))).to.be.eq("0x1626ba7e")
         })
+
+        it('should pass a keccak256 hash of the pre-image message to support contract signatures', async () => {
```

Comment:
Is it possible to add a negative test for this (so a test that fails) :thinking: 

I guess not ... but maybe you have an idea

Reply:
no, I don't either..